frontend/document/credocument.lua

```diff
@@ -1137,20 +1140,14 @@ function CreDocument:setupCallCache()
     end
 
     local no_op = function() end
-    local getTime = no_op
     local addStatMiss = no_op
     local addStatHit = no_op
     local dumpStats = no_op
     if do_stats then
         -- cache statistics
         self._call_cache_stats = {}
-        local _gettime = require("ffi/util").gettime
-        getTime = function()
-            local secs, usecs = _gettime()
-            return secs + usecs/1000000
-        end
         addStatMiss = function(name, starttime, not_cached)
-            local duration = getTime() - starttime
+            local duration = FFIUtil.getTimestamp() - starttime
```

Comment:
You could also switch them to use  `ffiUtil.getDuration(starttime)` like I started doing in:
https://github.com/koreader/koreader/blob/078b579081a29d24f229d87b08701ca6985a75fd/frontend/apps/reader/readerui.lua#L284-L286

Reply:
Yeah, I kept them separate mostly to avoid the noise of the time it'll take for string formatting, which is likely irrelevant in the grand scheme of things, but, oh, well ;p.